src/Misc/layoutbin/en-US/strings.json

```diff
@@ -170,6 +176,7 @@
   "Prompt0Default1": "Enter {0} (press enter for {1})",
   "RegisterAgentSectionHeader": "Register Agent",
   "Replace": "replace? (Y/N)",
+  "ResultsCommandNotFound": "##vso[results.{0}] is not a recognized command for Task command extension. TODO: DOC aka link",
```

Comment:
I prefixed all RM extension related string with "RM", May be its irrelevant. But it will be easier to identity string which were added for RM. Incase if I ever wanted to move the string to different place, it will be useful. Also we sort the string alphabetically, hence all RM strings are sorted out together. I don't know if everybody will agree to this. It was my choice.